package.json

```diff
@@ -23,5 +23,11 @@
   "bugs": {
     "url": "https://github.com/purtuga/esm-webpack-plugin/issues"
   },
-  "homepage": "https://github.com/purtuga/esm-webpack-plugin#readme"
+  "homepage": "https://github.com/purtuga/esm-webpack-plugin#readme",
+  "peerDependencies": {
+    "webpack": ">= 1.0.0"
```

Comment:
I'm thinking we should make this v4 instead of 1. Truth is I have only tested with v4 and I think the plugin's code is not compatible with earlier versions of webpack.